requirements-dev.txt

```
@@ -1 +1 @@
-flake8<3.4.0,>=3.3.0
+flake8>=3.5
```

I'm fine with making our requirement float more but can we lock this down to `<4.0`? Usually a major revision bump means breaking changes =/

```suggestion
flake8<4.0,>=3.5
```